app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt

```diff
@@ -28,6 +29,8 @@ class ModelSelectionActivity : BaseActivity(), ModelSelectionContract.View {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_model_selection)
 
+        startService(RemoteModelService.create(this))
```

Comment:
Shouldn't we hold a reference to RemoteModelService.create(this) here and use this reference onDestroy(),  instead of calling RemoteModelService.create(this) again there?